.github/workflows/ci-amd.yml

```diff
@@ -804,16 +804,15 @@ jobs:
 
   tests-go-sdk:
     name: "Go SDK tests"
-    needs: [build-info, build-ci-images]
+    needs: [build-info]
     runs-on: ${{ fromJSON(needs.build-info.outputs.amd-runners) }}
     timeout-minutes: 15
     permissions:
       contents: read
       packages: read
     if: >
       ( needs.build-info.outputs.run-go-sdk-tests == 'true' ||
-      needs.build-info.outputs.run-tests == 'true' &&
-      needs.build-info.outputs.only-new-ui-files != 'true')
+        needs.build-info.outputs.only-new-ui-files != 'true')
```

Comment:
So we need this one? I don't think so (going by its name)

Reply:
Actually, I found a lot more inconsistencies (historically) and I decided to fix them all - this output is completely removed now.